### ISSUE TITLE
main.CRITICAL in system.log

### ISSUE THREAD

**Issue by simkea** [magento/magento2#3075]
My ENV:
Debian Jessie, PHP 5.6.14, Apache, MariaDB 10.1.10, Magento CE 2.0.0 installed with GIT and Composer

Here is my shell-script for re-compile the mage2 stuff if i have changed my module:

```
#!/bin/bash
#################################################################################################################################

T_VERSION=mage2
T_STARTFOLDER=/var/www
T_RELPATH=${T_VERSION}-shop/magento2

#################################################################################################################################

echo "Cleaning"

rm -rf ${T_STARTFOLDER}/${T_RELPATH}/var/view_preprocessed/
rm -rf ${T_STARTFOLDER}/${T_RELPATH}/pub/static/adminhtml
rm -rf ${T_STARTFOLDER}/${T_RELPATH}/var/page_cache
rm -rf ${T_STARTFOLDER}/${T_RELPATH}/var/generation
rm -rf ${T_STARTFOLDER}/${T_RELPATH}/var/di
rm -rf ${T_STARTFOLDER}/${T_RELPATH}/var/cache

echo "re-compile"

sudo -u www-data ${T_STARTFOLDER}/${T_RELPATH}/bin/magento setup:upgrade
sudo -u www-data ${T_STARTFOLDER}/${T_RELPATH}/bin/magento setup:di:compile
sudo -u www-data ${T_STARTFOLDER}/${T_RELPATH}/bin/magento setup:static-content:deploy

echo "CHANGE permissions"
find ${T_STARTFOLDER}/${T_RELPATH} -type d -exec chmod 770 {} \;
find ${T_STARTFOLDER}/${T_RELPATH} -type f -exec chmod 660 {} \;
chmod u+x ${T_STARTFOLDER}/${T_RELPATH}/bin/magento

chown -R www-data.www-data ${T_STARTFOLDER}/${T_RELPATH}/

chmod 777 ${T_STARTFOLDER}/${T_RELPATH}/var
chmod -R 777 ${T_STARTFOLDER}/${T_RELPATH}/pub/media
chmod -R 777 ${T_STARTFOLDER}/${T_RELPATH}/pub/static
chmod 777 ${T_STARTFOLDER}/${T_RELPATH}/app/etc

chmod +x ${T_STARTFOLDER}/${T_RELPATH}/*.sh
chown root.root ${T_STARTFOLDER}/${T_RELPATH}/*.sh

echo "READY"
exit
```

Here can you see the output from shell-script:

Cache cleared successfully
File system cleanup:
/var/www/kaphingst/mage2-shop/magento2/var/generation/Composer
/var/www/kaphingst/mage2-shop/magento2/var/generation/Magento
/var/www/kaphingst/mage2-shop/magento2/var/generation/Symfony
The directory '/var/www/kaphingst/mage2-shop/magento2/var/di/' doesn't exist - skipping cleanup
/var/www/kaphingst/mage2-shop/magento2/pub/static/_requirejs
/var/www/kaphingst/mage2-shop/magento2/pub/static/deployed_version.txt
/var/www/kaphingst/mage2-shop/magento2/pub/static/frontend
Updating modules:
Module 'Magento_Wishlist':
Schema post-updates:
Module 'Magento_Store':
[...]
Module 'Magento_Wishlist':
Data install/update:
Module 'Magento_Store':
[...]
Module 'Magento_Wishlist':
Schema creation/updates:
Module 'Magento_Store':
[...]
Module 'Magento_Wishlist':
Please re-run Magento compile command
Compilation was started.
Interception cache generation... 7/7 [============================] 100% 5 mins 367.8 MiB
Generated code and dependency injection configuration successfully.
Requested languages: en_US
=== frontend -> Magento/blank -> en_US ===
[......]
Module 'Magento_Wishlist':
Successful: 1984 files; errors: 0
=== frontend -> Magento/luma -> en_US ===
[......]
Successful: 2058 files; errors: 0
=== adminhtml -> Magento/backend -> en_US ===
Successful: 1861 files; errors: 0
=== Minify templates ===
[......]
Successful: 850 files modified
New version of deployed files: 1453453115

the backend is running with no errors, but the system.log shows many main.INFO and main.CRITICAL

```
[2016-01-22 09:06:57] main.INFO: Cache file with merged layout: LAYOUT_adminhtml_STORE1_391fccd54da687afc76a5d6559b714906: Please correct the XML data and try again.  [] []

[2016-01-22 09:06:57] main.INFO: Cache file with merged layout: LAYOUT_adminhtml_STORE1_3cf9ab1b1723bedc1dcd665a812b00335: Please correct the XML data and try again.  [] []

[2016-01-22 09:06:57] main.CRITICAL: Broken reference: the 'header' tries to reorder itself towards 'global.notices', but their parents are different: 'page.wrapper' and 'notices.wrapper' respectively. [] []

[2016-01-22 09:06:57] main.CRITICAL: Broken reference: the 'page.breadcrumbs' tries to reorder itself towards 'notifications', but their parents are different: 'page.wrapper' and 'notices.wrapper' respectively. [] []

[2016-01-22 09:06:57] main.CRITICAL: Broken reference: the 'notification.messages' tries to reorder itself towards 'user', but their parents are different: 'header.inner.right' and 'header' respectively. [] []

[2016-01-22 09:06:58] main.INFO: Cache file with merged layout: LAYOUT_adminhtml_STORE1_36f1b068ec7ccf4878f9284dd1137afd1: Please correct the XML data and try again.  [] []

[2016-01-22 09:06:58] main.INFO: Item Magento_Backend::system_currency was removed [] []

[2016-01-22 09:06:58] main.INFO: Add of item with id Magento_AdminNotification::system_adminnotification was processed [] []

[2016-01-22 09:06:58] main.INFO: Add of item with id Magento_Theme::system_design_theme was processed [] []
[...]
[...]
[...]

[2016-01-22 09:06:58] main.INFO: Add of item with id Magento_Paypal::paypal_billing_agreement was processed [] []
```

**Comment by mazhalai** [comment]
@simkea Have you tried on the latest develop branch? Looks like it has been fixed in scope of #1795

**Comment by simkea** [comment]
Okay i pulled the last develop and now is the log clean.
